### DISCUSSION TITLE
Sign a message is not working when it includes empty spaces

### DISCUSSION THREAD

**Discussion by AltCoiFish** [DeFiCh/app#776] [Ideas]
Based on https://github.com/DeFiCh/app/issues/763

**What happened:**

Using the version v.2.3.3 and the CLI within the DFI-Wallet:

```signmessage "8xxxxx" "dfip-1 no"```

its not working and it falls back to the man help page of signmessage.

but

when I try

```signmessage "8xxxxx" "dfip-1-no"```

its working fine

same problem also for verifying any messages , if it includes a space its not working.
I also testes single-quotes 'it's also not working.

**Comment by izzycsy** [comment]
Hi @AltCoiFish , thanks for your feedback. The team will look into this.
